tools/build_defs/configure_script.bzl

```diff
@@ -7,8 +7,7 @@ def _pkgconfig_script(ext_build_dirs):
     script = []
     for ext_dir in ext_build_dirs:
         script.append("##increment_pkg_config_path## $$EXT_BUILD_DEPS$$/" + ext_dir.basename)
-
-    script.append("echo \"PKG_CONFIG_PATH=$${PKG_CONFIG_PATH:-}$$\"")
+        script.append("echo \"PKG_CONFIG_PATH=$${PKG_CONFIG_PATH:-}$$\"")
```

Comment:
nit: Do we really need to echo PKG_CONFIG_PATH for each entry in `ext_build_dirs`?

Reply:
Ah, good point. I just didn't want it echoing unnecessarily

Reply:
Fixed.